src/Administration/Controller/AdminAuthController.php

```diff
@@ -0,0 +1,64 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\Controller;
+
+use League\OAuth2\Server\AuthorizationServer;
+use Shopware\Administration\Login\Config\LoginConfigService;
+use Shopware\Administration\Login\Exception\LoginException;
+use Shopware\Administration\Login\LoginResponseService;
+use Shopware\Administration\Login\StateValidator;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+/**
+ * @internal
+ */
+#[Route(defaults: ['_routeScope' => ['api']])]
+#[Package('after-sales')]
```

Comment:
Additional question from my side:
@keulinho is the question of who will maintain this code already answered and can we adjust all the code annotations before merging it (so we don't forget anything)? Will it be `framework`? 🙈